### ISSUE TITLE
[Bug] wrioting to csv 

### ISSUE THREAD

**Issue by bijanfallah** [marians/dwd-weather#10]
type: bug 
ubuntu18.04; python3.6.9 
after running these two commands:

```
dwdweather stations --resolution=daily
dwdweather stations --type csv --file stations.csv
```

I get the following error:

2020-06-10 18:05:28,488 [dwdweather.client   ] INFO   : Acquiring dataset for resolution "hourly" from "https://opendata.dwd.de/climate_environment/CDC/observations_germany/climate/hourly"
2020-06-10 18:05:28,488 [dwdweather.core     ] INFO   : Using cache database /home/bkd/.dwd-weather/dwdweather2.db
Traceback (most recent call last):
  File "/usr/local/bin/dwdweather", line 10, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.6/dist-packages/dwdweather/commands.py", line 182, in run
    args.func(args)
  File "/usr/local/lib/python3.6/dist-packages/dwdweather/commands.py", line 46, in get_stations
    f.write(output)
TypeError: a bytes-like object is required, not 'str'

**Comment by amotl** [comment]
Dear Bijan,

does the problem also happen with the successor of this library, dwdweather2 [1]?

With kind regards,
Andreas.

[1] https://github.com/panodata/dwdweather2

**Comment by amotl** [comment]
Dear Bijan,

I checked myself and indeed the error did also happen there. However, I just fixed it through `dwdweather2==0.14.0`.

With kind regards,
Andreas.

**Comment by bijanfallah** [comment]
Perfect. The `dwdweather2==0.14.0` did the job.
